### PR TITLE
9508 - Update caseless petition

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

**Review by shumway-tm — APPROVED** [review]

File: web-client/src/styles/custom.scss [hunk 1]
```diff
@@ -1984,3 +1984,17 @@ button.change-scanner-button {
   background-size: 30% 30%;
   outline: 3px solid #005ea2;
 }
+
+.text-right {
```
Comment (rachelschneiderman) [inline web-client/src/styles/custom.scss]: 🔧 There is already a text-right class in `utils.scss`!

File: web-client/src/styles/custom.scss [hunk 2]
```diff
@@ -1984,3 +1984,17 @@ button.change-scanner-button {
...
+  text-align: right;
+}
+
+.width-0 {
```
Comment (rachelschneiderman) [inline web-client/src/styles/custom.scss]: 🔧 I believe USWDS also defines this as a class!

File: web-client/src/styles/custom.scss (hunk 1 above)
Comment (codyseibert) [reply web-client/src/styles/custom.scss:1988]: the utils.scss is only used for the pdf html generators, so we also needed it here I think.

Comment (codyseibert) [reply web-client/src/styles/custom.scss]: I removed this class; we are not even using it at this point.  I think we accidentally committed it.

File: web-client/src/styles/custom.scss (hunk 2 above)
Comment (codyseibert) [reply web-client/src/styles/custom.scss]: ty! I removed this class; we are not even using it at this point.  I think we accidentally committed it.

**Review by mrinsin — APPROVED** [review]

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**